public/omni/infrastructure-and-extensions/infrastructure-providers.mdx

```diff
@@ -14,10 +14,14 @@ The current infrastructure providers are:
 
 * [Bare metal provider](https://github.com/siderolabs/omni-infra-provider-bare-metal)
 * [KubeVirt provider](https://github.com/siderolabs/omni-infra-provider-kubevirt)
+* [Proxmox provider](https://github.com/siderolabs/omni-infra-provider-proxmox)
+* [Oxide provider](https://github.com/oxidecomputer/omni-infra-provider-oxide)
```

Comment:
Can you split this out into it's own list like "3rd party providers" just so people know it's not created or maintained by us